server/channels/app/channel.go

```diff
@@ -3016,19 +2985,18 @@ func (a *App) MarkChannelsAsViewed(c request.CTX, channelIDs []string, userID st
 	}
 
 	if *a.Config().ServiceSettings.EnableChannelViewedMessages {
-		for _, channelID := range channelIDs {
-			message := model.NewWebSocketEvent(model.WebsocketEventChannelViewed, "", "", userID, nil, "")
-			message.Add("channel_id", channelID)
-			a.Publish(message)
-		}
+		message := model.NewWebSocketEvent(model.WebsocketEventMultipleChannelsViewed, "", "", userID, nil, "")
```

Comment:
The channels viewed websocket event was being skipped if the websocket system was too busy. Should this one be added to that too?

Reply:
If this is the event we're using for the regular single view of a channel, then yes, we should add it.